nifi-nar-bundles/nifi-extension-utils/nifi-event-transport/src/main/java/org/apache/nifi/event/transport/netty/NettyEventSenderFactory.java

```diff
@@ -116,6 +117,8 @@ public void setMaxConnections(final int maxConnections) {
         bootstrap.remoteAddress(new InetSocketAddress(address, port));
         final EventLoopGroup group = getEventLoopGroup();
         bootstrap.group(group);
+        bootstrap.option(ChannelOption.SO_SNDBUF, FRAME_SIZE);
+        bootstrap.option(ChannelOption.SO_RCVBUF, FRAME_SIZE);
```

Comment:
Is this necessary for all uses of the Sender Factory?  Changing the value to a nullable Integer would make it more configurable if it can be optional.